source/Octopus.Tentacle/Variables/EnvironmentVariables.cs

```diff
@@ -20,5 +20,6 @@ public static class EnvironmentVariables
         public const string TentacleExecutablePath = "TentacleExecutablePath";
         public const string TentacleProgramDirectoryPath = "TentacleProgramDirectoryPath";
         public const string AgentProgramDirectoryPath = "AgentProgramDirectoryPath";
+        public const string TentacleEnableAsyncHalibut = "TentacleEnableAsyncHalibut";
```

Comment:
I don't love the naming scheme here... `OCTOPUS__Tentacle__EnableAsyncHalibut` probably fits with our current approach, but I wasn't sure if that was Server-specific